### ISSUE TITLE
[Accessibility] Accessibility focus moves inappropriatley to the code editor section.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227422]
- VS Code Version: 1.88.0

- OS Version: MacOS

**Issue**: When loading the VSCode iframe in Lambda Console, the accessibility focus moves inappropriately to the code editor section, instead of top of the page.

**Impact**: Screen reader users and screen magnification users may become disoriented. Keyboard users will have to navigate through the contents of the page.

**Steps to Reproduce:**

- Have the accessibility screen reader open.
- On a webpage load with VSCode iframe loading from a CDN
- The screen reader focus automatically moves inappropriately to the code editor section instead of top of page.
- VSCode is somewhere intercepting the accessibility focus to go straight to editor section instead of top of page. We don’t have this issue currently with Cloud9 in lambda console.

**[Recommendation]**
 Ensure content updates define focus updates appropriately. When a link is activated, focus should be on the top of the page. Use the JavaScript focus() method to move keyboard focus to this element.

**Comment by meganrogge** [comment]
Thanks for the issue, where is this reproducing? Including @mjbvz because of the iframe component

**Comment by meganrogge** [comment]
We intentionally focus the code editor, thus closing.

**Comment by Krituhar** [comment]
Could you point to me where in your code base do you do that ? Trying to understand the logic.
